### DISCUSSION TITLE
What is the purpose/benefit of the @tailwind directive?

### DISCUSSION THREAD

**Discussion by browner12** [tailwindlabs/tailwindcss#2735] [Help]
As best I can tell, `@tailwind` is serving the same purpose as `@import`. Maybe it's doing some auto path resolution? Maybe it's auto appending the 'screen' section? It's hard to tell, and it's causing some confusion.

Could we get some clarification, and maybe add an explanation of the differences between the two to the docs?

Thanks!

Reference File:

https://github.com/tailwindlabs/tailwindcss/blob/master/src/lib/substituteTailwindAtRules.js

**Comment by erhankaradeniz** [comment]
You can find some explanation in the installation guide on why those two exist: [https://tailwindcss.com/docs/installation#add-tailwind-to-your-css](https://tailwindcss.com/docs/installation#add-tailwind-to-your-css)

**Comment by browner12** [comment]
Thanks, but I was hoping for a more in-depth explanation than that.
